src/keyring/KeyringController.test.ts

```diff
@@ -45,7 +46,7 @@ describe('KeyringController', () => {
     keyringTypes: string[];
     keyrings: Keyring[];
   };
-  const additionalKeyrings = [QRKeyring];
+  const additionalKeyrings = [QRKeyring, LedgerKeyring];
```

Comment:
This is kind of a breaking change. Once we have the new controller package referenced in the mobile app the LedgerKeyring needs to be set as an additional Keyring. To be fair, this is the same requirement as the already existing QRKeyring has.